File: Eclipse/Jumpy/src/com/example/jumpy/Highscore.java
package com.example.jumpy;

public class Highscore
{
	private int player_id;
	private int score;
	private int height;
	
	public Highscore(int player_id, int score, int height)
	{
		this.player_id = player_id;
		this.score = score;
		this.height = height;
	}

	public int getPlayer_id()
	{
		return player_id;
	}

	public int getScore()
	{
		return score;
	}

	public int getHeight()
	{
		return height;
	}
	
}

File: Unity/Jumpy/Assets/Scripts/HighScore.cs
using UnityEngine;
using System.Collections;

public class HighScore 
{
	private int height;
	private string name;

	public HighScore(string name, int height)
	{
		this.name = name;
		this.height = height;
	}

	public int getHeight()
	{
		return height;
	}

	public string getName()
	{
		return name;
	}
}
